application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -180,6 +213,39 @@ public void buildGraph() {
             }
           }
         }
+        // Calculate transfers between stops that can use trips with cars if configured.
+        if (carsAllowedStops.contains(stop)) {
+          for (RouteRequest transferProfile : carsAllowedStopTransferRequests) {
+            StreetMode mode = transferProfile.journey().transfer().mode();
+            for (NearbyStop sd : carsAllowedStopNearbyStopFinders
+              .get(mode)
+              .findNearbyStops(ts0, transferProfile, transferProfile.journey().transfer(), false)) {
+              // Skip the origin stop, loop transfers are not needed.
+              if (sd.stop == stop) {
```

Comment:
Isn't this block a copy&paste job from the one above? Can it be reused?

Reply:
I created a method for creating the PathTransfer and TransferKey. Is this fine?